### ISSUE TITLE
Workspace creation failed with setting it's name only

### ISSUE THREAD

**Issue by jgueytat** [eclipse-che/che#19166]
### Describe the bug
The creation of an empty workspace fails with the error : **this.websocketStream is undefined**.

### Steps to reproduce
I followed the [installing che on minikube tutorial](https://www.eclipse.org/che/docs/che-7/installation-guide/installing-che-on-minikube/).
The extra stuff that I did were :   
- change the **permission** of the **/usr/local/lib/chectl** folder
- complete my **/etc/hosts** file with the following lines

```
192.168.39.28   che-eclipse-che.192.168.39.28.nip.io
192.168.39.28   plugin-registry-eclipse-che.192.168.39.28.nip.io
192.168.39.28   devfile-registry-eclipse-che.192.168.39.28.nip.io
```

### Runtime

   - [x] minikube (include output of `minikube version` and `kubectl version`)

```
$ minikube version
minikube version: v1.17.1
commit: 043bdca07e54ab6e4fc0457e3064048f34133d7e
```
```
$ kubernetes version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.4", GitCommit:"e87da0bd6e03ec3fea7933c4b5263d151aafd07c", GitTreeState:"clean", BuildDate:"2021-02-18T16:12:00Z", GoVersion:"go1.15.8", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-13T13:20:00Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
```

### Screenshots

![che-01](https://user-images.githubusercontent.com/1152718/109425008-8540fd80-79e6-11eb-94a9-9e23d5d8ec4a.png)
![che-02](https://user-images.githubusercontent.com/1152718/109425012-88d48480-79e6-11eb-9e83-655886906d82.png)

### Installation method

```sh
$ chectl server:deploy --platform minikube --installer helm
```

<details>
<summary>output</summary>
<p>

```
› Current Kubernetes context: 'minikube'
  ✔ Verify Kubernetes API...OK
  ✔ 👀  Looking for an already existing Eclipse Che instance
    ✔ Verify if Eclipse Che is deployed into namespace "eclipse-che"...it is not
  ✔ ✈️  Minikube preflight checklist
    ✔ Verify if kubectl is installed
    ✔ Verify if minikube is installed
    ✔ Verify if minikube is running
    ↓ Start minikube [skipped]
      → Minikube is already running.
    ✔ Check Kubernetes version: Found v1.20.2.
    ✔ Verify if minikube ingress addon is enabled
    ↓ Enable minikube ingress addon [skipped]
      → Ingress addon is already enabled.
    ✔ Retrieving minikube IP and domain for ingress URLs...192.168.39.28.nip.io.
    ✔ Checking minikube version... 1.17.1
    ↓ Check if cluster accessible [skipped]
  ✔ Following Eclipse Che logs
    ↓ Start following Operator logs [skipped]
    ✔ Start following Eclipse Che server logs...done
    ✔ Start following Postgres logs...done
    ✔ Start following Keycloak logs...done
    ✔ Start following Plugin registry logs...done
    ✔ Start following Devfile registry logs...done
    ✔ Start following namespace events...done
 ›   Warning: Consider using the more reliable 'OLM' installer when deploying a stable release of Eclipse Che (--installer=olm).
  ✔ 🏃‍  Running Helm to install Eclipse Che
    ✔ Verify if helm is installed
    ✔ Check Helm Version: Found v3.5.2+g167aac7
    ✔ Create Namespace (eclipse-che)...does already exist.
    ✔ Check Eclipse Che TLS certificate...self-signed TLS certificate secret found
    ✔ Check Cluster Role Binding...does not exists.
    ✔ Updating Helm Chart dependencies...done.
    ✔ Deploying Eclipse Che Helm Chart...done.
  ✔ ✅  Post installation checklist
    ✔ Devfile registry pod bootstrap
      ✔ Scheduling...done
      ✔ Downloading images...done
      ✔ Starting...done
    ✔ Plugin registry pod bootstrap
      ✔ Scheduling...done
      ✔ Downloading images...done
      ✔ Starting...done
    ✔ Eclipse Che pod bootstrap
      ✔ Scheduling...done
      ✔ Downloading images...done
      ✔ Starting...done
    ✔ Eclipse Che status check...done
  ✔ Prepare post installation output...done
  ✔ Show important messages
    ✔ Eclipse Che 7.26.2 has been successfully deployed.
    ✔ Documentation             : https://www.eclipse.org/che/docs/
    ✔ -------------------------------------------------------------------------------
    ✔ Users Dashboard           : https://che-eclipse-che.192.168.39.28.nip.io
    ✔ Admin user login          : "admin:admin". NOTE: must change after first login.
    ✔ -------------------------------------------------------------------------------
    ✔ Plug-in Registry          : https://plugin-registry-eclipse-che.192.168.39.28.nip.io/v3
    ✔ Devfile Registry          : https://devfile-registry-eclipse-che.192.168.39.28.nip.io
    ✔ -------------------------------------------------------------------------------
    ✔ -------------------------------------------------------------------------------
Command server:deploy has completed successfully in 00:38.
```

</p>
</details>

```sh
$ chectl version
chectl/7.26.2 linux-x64 node-v12.20.2
```

### Environment

   - [x] my computer
       - [x] Linux

### Eclipse Che Logs

[eclipse-che.zip](https://github.com/eclipse/che/files/6057119/eclipse-che.zip)

### Additional context

![che-03](https://user-images.githubusercontent.com/1152718/109425296-e917f600-79e7-11eb-9666-8b81b17f529a.png)

**Comment by sleshchenko** [comment]
I tried Che on minikube with operator with multi-host and single-host, and both work fine, no issue with empty workspace creation.

@jgueytat Could you elaborate what you tried to solve with modifying /etc/hosts?
Do you have issue only with empty workspace? If you try sample on Get Started page or choose templates on Create Custom Workspace then it works fine?

**Comment by jgueytat** [comment]
DNS rebinding is not usable for me. I can not use this service.
No I already tried to use a template and got the same result.

**Comment by jgueytat** [comment]
I get the same error message.
![WithDevfile](https://user-images.githubusercontent.com/1152718/109778936-b64b4900-7c05-11eb-91b3-0ae1e8b8f68d.png)

**Comment by sleshchenko** [comment]
Ah. I assume it duplicates https://github.com/eclipse/che/issues/19038 and should be fixed for 7.27.0.
To test it you can use the `nightly` image for che server: `kubectl edit deployment che -n eclipse-che`.

**Comment by jgueytat** [comment]
Yes I got the exact same message on error while trying with another IDE ! Good catch !
